internal/pkg/term/selector/local_selector.go

```diff
@@ -177,7 +239,7 @@ func (s *localFileSelector) Dockerfile(selPrompt, notFoundPrompt, selHelp, notFo
 	return sel, nil
 }
 
-// ListDockerfiles returns the list of Dockerfiles within the current
+// listDockerfiles returns the list of Dockerfiles within the current
```

Comment:
totally unrelated and not for this pr but this func could probably use your `getDirAndFileNames` func now!

Reply:
Oh, good call!